package.json

```diff
@@ -12,6 +12,7 @@
     "url": "https://github.com/kristerkari/stylelint-scss/issues"
   },
   "dependencies": {
+    "lodash": "^4.11.1",
```

Comment:
It looks like this is not being used anywhere? Let's not add lodash dependency unless it is really needed.